frontend/src/components/common/table/Table.tsx

```diff
@@ -11,6 +11,7 @@ import {
 } from "@chakra-ui/react";
 
 import Pagination from "./Pagination";
+import usePaginatedData from "./usePaginatedData";
```

Comment:
can we remove `useState` from line 1 import!

Reply:
oops yes